Code/Framework/AzFramework/AzFramework/Entity/SliceEntityOwnershipService.cpp

```diff
@@ -146,11 +143,9 @@ namespace AzFramework
     {
         AZ_Assert(m_rootAsset && m_rootAsset->GetComponent(), "Root slice has not been created.");
         AZ_Assert(m_entitiesRemovedCallback, "Callback function for DestroyEntityById has not been set.");
+        SliceEntityRequestBus::MultiHandler::BusDisconnect(entityId);
         m_entitiesRemovedCallback({ entityId });
-
-        // Entities removed through the application (as in via manual 'delete'),
-        // should be removed from the root slice, but not again deleted.
-        return m_rootAsset->GetComponent()->RemoveEntity(entityId, false);
+        return m_rootAsset->GetComponent()->RemoveEntity(entityId);
```

Comment:
This changes the behavior because previously this function didn't delete the entity but now it will because RemoveEntity() takes 'true' as the default argument. With the above changes, I think it is fine to delete the entity here. Just keep an eye on this because if any bugs raise because of this we can always flip it back to 'false' again.

Reply:
Yeah, I will keep an eye on it. It makes sense to delete the entity here especially because in SliceEntityOwnershipService.h it reads:
`        //! Deletes the entity id from the root-slice and destroys it.
        //! @param entityId
        bool DestroyEntityById(AZ::EntityId entityId) override;`